style.css

```diff
@@ -67,3 +67,14 @@ p {
     padding-top: 10px;
     padding-left: 20px;
 }
+
+/*
+ * News
+ */
+
+.post {
+    margin-bottom: 12px;
+}
+.post-title {
+    font-size: 30px;
```

Comment:
I would really like if we didn't add any extra font sizes.  Linters start complaining once you add too many.  `h2` is already 30px.